README.md

```diff
@@ -170,12 +170,137 @@ Note that these transitions are just meant to redirect calls to the correspondin
+4. Have one of the owners of the old contract invoke the `ExecuteTransaction` transition on the old contract. This will cause the entire balance of the old contract to be transferred to the new wallet. Note that no un-executed transactions will be transferred to the new wallet along with the funds.
+
+WARNING:
+
+If a sufficient number of owners lose their private keys, or for any other reason are unable or unwilling to sign for new transactions, the funds in the wallet will be locked forever. It is therefore a good idea to set required_signatures to a value strictly less than the number of owners, so that the remaining owners can retrieve the funds should such a scenario occur.
+
+If an owner loses his private key, the remaining owners should move the funds to a new wallet (using the workflow described above) to  ensure that funds are not locked if another owner loses his private key. The owner who originally lost his private key can generate a new key, and the corresponding address be added to the new wallet, so that the same set of persons own the new wallet.
+
+### Multi-signature contract Roles and Privileges
+
+| Name | Description & Privileges |
+|--|--|
+|`owners` | The users who own this contract. |
+|`initiator`| The user who calls the multi-signature contract. |
+
+### Immutable Parameters
+
+The table below lists the parameters that are defined at the contract deployment time and hence cannot be changed later on.
+
+| Name | Type | Description |
+|--|--|--|
+|`owners_list`| `List ByStr20` | List of init owners. |
+|`required_signatures`| `Uint32` | Minimum amount of signatures to execute a transaction. |
+
+### Mutable Fields
+
+The table below presents the mutable fields of the contract and their initial values.
+
+| Name | Type | Initial Value | Description |
+|--|--|--|--|
+|`owners`| `Map ByStr20 Bool` | `mk_owners_map owners_list` | Map of owners. |
+|`transactionCount`| `Uint32` | `0` | The count of transactions have been executed by this contract. |
+|`signatures`| `Map Uint32 (Map ByStr20 Bool)` | `Emp Uint32 (Map ByStr20 Bool)` | Collected signatures for transactions by transaction ID. |
+|`signature_counts`| `Map Uint32 Uint32` | `Emp Uint32 Uint32` | Running count of collected signatures for transactions. |
+|`transactions`| `Map Uint32 Transaction` | `Emp Uint32 Transaction` | Transactions have been submitted but not exected yet. |
+
+### Transitions
+
+All the transitions in the contract can be categorized into three categories:
+- _submit transitions_ create transactions for future signoff.
+- _action transitions_ let owners sign, revoke or execute submitted transactions.
+- _callback transitions_ are called by the proxy contract. No action required, but must be defined, since to the wallet will fail otherwise.
+- `AddFunds transition` are use for adding native funds(ZIL) to the wallet.
```

Comment:
Add a little bit more explanation here, that the `accept` keyword is a reserved word for that, without which this smart contract will not be able to hold native funds